### PR TITLE
NO-DED adding x-agent-id to Web and RTM Agent Chat API

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

As agreed with Paweł, the new Bot section has been added to the Web and RTM Agent Chat API. The section is needed as many devs ask about it or have problems with calling methods as bots.

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Review by litwinow — APPROVED** [review]

File: src/pages/messaging/agent-chat-api/index.mdx [hunk 1]
```diff
@@ -66,6 +66,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/index.mdx:71]: Not sure if "call an event" is the best collocation. Maybe "to do it", which will refer to sending an event from the previous sentence?

File: src/pages/messaging/agent-chat-api/index.mdx [hunk 2]
```diff
@@ -66,6 +66,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `X-Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/index.mdx:73]:
```suggestion
In both situations, the `X-Author-Id` header is what you need to include in your request. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```

Comment (ale-kacperczyk) [reply src/pages/messaging/agent-chat-api/index.mdx:73]: I imagined the header with hands :P I know it's not literal, but still

File: src/pages/messaging/agent-chat-api/index.mdx [hunk 3]
```diff
@@ -66,6 +66,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
...
+
+`X-Author-Id: <bot_id>` header is needed with a [bearer token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/index.mdx:75]:
```suggestion
`X-Author-Id: <bot_id>` header is needed with a [Bearer Token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```

Comment (ale-kacperczyk) [reply src/pages/messaging/agent-chat-api/index.mdx:75]: We capitalize it in other docs

File: src/pages/messaging/agent-chat-api/index.mdx [hunk 4]
```diff
@@ -66,6 +66,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
```
Comment (ale-kacperczyk) [inline src/pages/messaging/agent-chat-api/index.mdx:69]: I'm not sure about the title of this section. I'd see something more specific, such as "Calling the API as a bot" or something like that? To inform that this section doesn't describe bots in general, but tackles a specific scenario.

Also, I think we should link this section [here](https://deploy-preview-1372--livechat-public-docs.netlify.app/messaging/agent-chat-api#methods) 👇
image

File: src/pages/messaging/agent-chat-api/index.mdx (hunk 1 above)
Comment (kamilatokarska) [reply src/pages/messaging/agent-chat-api/index.mdx:71]: You are right; it sounds like a purple pose. I will change it.

Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/index.mdx:71]:
```suggestion
Sometimes, you want to [send an event as a bot](#send-event). To do it, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

File: src/pages/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 5]
```diff
@@ -93,6 +93,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/rtm-reference/index.mdx:98]:
```suggestion
Sometimes, you want to [send an event as a bot](#send-event). To do it, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

File: src/pages/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 6]
```diff
@@ -93,6 +93,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/rtm-reference/index.mdx:100]:
```suggestion
In both situations, the `Author-Id` header is what you need to include in your request. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```

File: src/pages/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 7]
```diff
@@ -93,6 +93,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
...
+
+`Author-Id: <bot_id>` header is needed with a [bearer token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/rtm-reference/index.mdx:102]:
```suggestion
`Author-Id: <bot_id>` header is needed with a [Bearer Token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```

File: src/pages/messaging/agent-chat-api/v3.4/index.mdx [hunk 8]
```diff
@@ -71,6 +71,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `X-Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.4/index.mdx:78]:
```suggestion
In both situations, the `X-Author-Id` header is what you need to include in your request. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```

File: src/pages/messaging/agent-chat-api/v3.4/index.mdx [hunk 9]
```diff
@@ -71,6 +71,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.4/index.mdx:76]:
```suggestion
Sometimes, you want to [send an event as a bot](#send-event). To do it, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

File: src/pages/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx [hunk 10]
```diff
@@ -98,6 +98,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx:103]:
```suggestion
Sometimes, you want to [send an event as a bot](#send-event). To do it, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

File: src/pages/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx [hunk 11]
```diff
@@ -98,6 +98,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
+
+`Author-Id: <bot_id>` header is needed with a [bearer token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.4/rtm-reference/index.mdx:107]:
```suggestion
`Author-Id: <bot_id>` header is needed with a [Bearer Token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```

File: src/pages/messaging/agent-chat-api/v3.6/index.mdx [hunk 12]
```diff
@@ -67,6 +67,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `X-Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.6/index.mdx:74]:
```suggestion
In both situations, the `X-Author-Id` header is what you need to include in your request. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```

File: src/pages/messaging/agent-chat-api/v3.6/index.mdx [hunk 13]
```diff
@@ -67,6 +67,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.6/index.mdx:72]:
```suggestion
Sometimes, you want to [send an event as a bot](#send-event). To do it, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

File: src/pages/messaging/agent-chat-api/v3.6/rtm-reference/index.mdx [hunk 14]
```diff
@@ -94,6 +94,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.6/rtm-reference/index.mdx:101]:
```suggestion
In both situations, the `Author-Id` header is what you need to include in your request. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
```

File: src/pages/messaging/agent-chat-api/v3.6/rtm-reference/index.mdx [hunk 15]
```diff
@@ -94,6 +94,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.6/rtm-reference/index.mdx:99]:
```suggestion
Sometimes, you want to [send an event as a bot](#send-event). To do it, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
```

File: src/pages/messaging/agent-chat-api/v3.6/rtm-reference/index.mdx [hunk 16]
```diff
@@ -94,6 +94,14 @@ The `filters`, `limit`, and `sort_order` parameters can't be provided along with
 
 💡 The maximum duration of the `page_id` parameter before it expires is one month.
 
+## Bots
+
+Sometimes, you want to [send an event as a bot](#send-event). To call an event, you need to have a token that allows you to send it. The token can be easily generated for an agent, but you cannot generate a token for a bot. The other time you want to [add a user to your chat](#add-user-to-chat), and this user happens to be a bot in the group with no access to this chat, so you cannot add it.  
+
+In both situations, the `Author-Id` header comes with a helping hand. With it, you can authorize using the agent’s token and send the event as a bot or add a bot to a chat even if the bot belongs to the group that has no access to this chat.
+
+`Author-Id: <bot_id>` header is needed with a [bearer token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```
Comment (kamilatokarska) [inline src/pages/messaging/agent-chat-api/v3.6/rtm-reference/index.mdx:103]:
```suggestion
`Author-Id: <bot_id>` header is needed with a [Bearer Token](/authorization/agent-authorization/) that needs to have the same `client_id` as the bot or [PAT](/authorization/agent-authorization#personal-access-tokens) that is from the organization that owns the bot's `client_id`.
```

File: src/pages/messaging/agent-chat-api/index.mdx (hunk 4 above)
Comment (kamilatokarska) [reply src/pages/messaging/agent-chat-api/index.mdx:69]: Yup, a specific title will be much better now that you have written it.

File: src/constant/index.js [hunk 17]
```diff
@@ -10,7 +10,7 @@ const VERSIONS_GROUPS = {
     STABLE_VERSION: "3.0",
     LEGACY_VERSIONS: ["1.0", "2.0"],
     DEV_PREVIEW_VERSION: "",
-    DEPRECATED_VERSIONS: ["3.1"],
+    DEPRECATED_VERSIONS: [""],
```
Comment (kamilatokarska) [inline src/constant/index.js:13]: as mentioned [here](https://text.slack.com/archives/D03919XPNNM/p1682698394034889), I checked and it looks like v3.1 should not be here, so I removed it.

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**